### ISSUE TITLE
TypeError: Cannot read property 'get' of undefined

### ISSUE THREAD

**Issue by lukeocodes** [netlify/netlify-cms#4480]
**Describe the bug**

I have enabled i18n for content authoring. I believe I should be able to save a post in the primary language without adding all the translations at the same time. When I try to save, I get no visible error. Instead, I get this error in the console.

**To Reproduce**

0. On a site configured as my config linked below
1. Go to `/admin`
2. Click on 'New Blog Post'
3. Complete the form for the English post
4. Try and click on 'Save'
5. See error in console

**Expected behavior**

0. On a site configured as my config linked below
1. Go to `/admin`
2. Click on 'New Blog Post'
3. Complete the form for the English post
4. Try and click on 'Save'
5. Saves only the translation you've created

**Screenshots**

![TypeError: Cannot read property 'get' of undefined](https://user-images.githubusercontent.com/956290/96470214-f517ce00-1225-11eb-8113-01a41d0d527a.png)

![Entire console output including TypeError](https://user-images.githubusercontent.com/956290/96470977-c0f0dd00-1226-11eb-9a5a-6906aeec427b.png)

**Applicable Versions:**

 - Netlify CMS: 2.10.63
 - Git provider: GitHub
 - OS: Mac
 - Browser version: Version 86.0.4240.80 (Official Build) (x86_64)

**CMS configuration**

https://github.com/Nexmo/deved-platform/blob/main/static/admin/config.yml

**Additional context**

If I complete all the fields for all the languages, it then lets me save!

**Comment by lukeocodes** [comment]
This error occurred in these other issues (#4327 #3772 #3690) but based on the config and the lack of errors I couldn't be sure if related. They're all closed without resolution too.

**Comment by frankspin89** [comment]
Yes, this error seems to be related only to creating new content without entering all translations. 

Versions: 
"netlify-cms-app": "^2.12.27",
git-gateway, local_backend
OS: Windows 10
Browser version: Edge Version 86.0.622.43 (Official build) (64-bit)

**app.js**
`// This global flag enables manual initialization.
window.CMS_MANUAL_INIT = true

import CMS from 'netlify-cms-app'

import { pages } from './collections/pages'


CMS.init({
  config: {
    load_config_file: false,
    local_backend: true,
    media_folder: '/static/meda',
    public_folder: '/meda',
    backend: {
      name: 'git-gateway'
    },
    media_library: {
      name: '',
      config: {
        max_file_size: 700000
      }
    },
    i18n: {
      structure: 'multiple_files',
      locales: ['nl', 'en', 'de'],
      default_locale: 'nl'
    },
    collections: [pages]
  },
})`

**Pages collection**

`import { image } from "../partials/image"

const fields = [
  {
    label: 'Title',
    name: 'title',
    widget: 'string',
    required: false,
    i18n: true
  },
  {
    label: 'Url',
    name: 'url',
    widget: 'string',
    required: false,
    i18n: true
  },
  image
]

const pages = {
  label: 'Pages',
  name: 'pages',
  create: true,
  i18n: true,
  media_folder: 'images',
  public_folder: 'images',
  format: 'frontmatter',
  path: '{{slug}}/index',
  editor: {
    preview: false
  },
  folder: 'content/pages',
  fields: [
    ...fields
  ],
}


export {
  pages
}`

**Image**
`export const image = {
  label: "Image",
  name: 'image',
  widget: 'object',
  i18n: true,
  fields: [
    {
      label: 'Source',
      name: 'source',
      widget: 'image',
      required: false,
      i18n: true
    },
    {
      label: 'Alt text',
      name: 'alt',
      widget: 'string',
      required: false,
      i18n: true
    },
  ]
}`

**Error on creating page**

`netlify-cms-app.js:167 Uncaught (in promise) TypeError: Cannot read property 'get' of undefined
    at eval (netlify-cms-app.js:167)
    at eval (netlify-cms-app.js:1)
    at it.__iterate (netlify-cms-app.js:1)
    at it.reduce (netlify-cms-app.js:1)
    at s (netlify-cms-app.js:167)
    at t.serializeValues (netlify-cms-app.js:167)
    at r (netlify-cms-app.js:34)
    at eval (netlify-cms-app.js:43)
    at Array.forEach (<anonymous>)
    at t.serializeI18n (netlify-cms-app.js:43)`

**Comment by lukeocodes** [comment]
Let me know if you need anything else, happy to help :)
